app/models/cinema_hall.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+class CinemaHall < ApplicationRecord
+    has_many :screenings
```

Comment:
It's a good practice to define `dependent` option on `has_many` relation, like: `has_many :screenings, dependent: :destroy` - the option value should depend on your needs. 

You can read more about the option (and possible values) in the [Rails documentation](https://api.rubyonrails.org/classes/ActiveRecord/Associations/ClassMethods.html#method-i-has_many).